site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-newSiteCourseManual.vm

```diff
@@ -224,6 +224,16 @@
 						value="$tlang.getString('gen.cancel')"
 						accesskey="x"
 						onclick="SPNR.disableControlsAndSpin( this, null );document.requestCourseForm.option.value='cancel'; document.requestCourseForm.submit(); return false;" />
+				#if( $!contNoRosterEnabled )
+				<input
+					type="button" 
+					accesskey="x"
+					name="Cancel" 
+					id="cancelButton" 
+					value="$tlang.getString('nscourse.no_rosters')"
+					onclick="SPNR.disableControlsAndSpin( this, null );document.requestCourseForm.option.value='norosters'; document.requestCourseForm.submit(); return false;"
```

Comment:
@bjones86 ,  thanks, but that was included in a previous patch on top of the original I believe.